src/access/wac.test.ts

```diff
@@ -884,3 +884,372 @@ describe("getGroupAccess", () => {
     });
   });
 });
+
+describe("getPublicAccess", () => {
+  it("calls the included fetcher by default", async () => {
+    const mockedFetcher = jest.requireMock("../fetcher.ts") as {
+      fetch: jest.Mock<
+        ReturnType<typeof window.fetch>,
+        [RequestInfo, RequestInit?]
+      >;
+    };
+
+    const resource = getMockDataset(
+      "https://some.pod/resource",
+      "https://some.pod/resource.acl"
+    );
+    await getPublicAccess(resource);
+
+    expect(mockedFetcher.fetch.mock.calls[0][0]).toEqual(
+      "https://some.pod/resource.acl"
+    );
+  });
+
+  it("returns null if no ACL is accessible", async () => {
+    const mockFetch = jest
+      .fn(window.fetch)
+      // No resource ACL available...
+      .mockResolvedValueOnce(
+        mockResponse("", {
+          status: 404,
+          url: "https://some.pod/resource.acl",
+        })
+        // Link to the fallback ACL...
```

Comment:
Just... What is going on here? :P

Reply:
I'm not sure if the answer is that something beyond my control is very stubborn, or if I'm very stupid :p

Reply:
To stupid, stupid and a half 9f91dfc5940a10ca94b3fd401dc6c96346a241a7